src/views/Player.vue

```diff
@@ -144,6 +150,7 @@ export default {
               },
               
               invertTime: false,
+              clickToPlay: false
```

Comment:
This option is configured in plyr as `false` because we want to record when they pause/play. And if we allow `clickToPlay`, we will not be able to record their actions, as tapping to play is allowed all over the screen.